lib/cli-engine.js

```diff
@@ -347,7 +346,7 @@ function getCacheFile(cacheFile, cwd) {
  */
 function CLIEngine(options) {
 
-    options = lodash.assign(Object.create(null), defaultOptions, options);
+    options = lodash.assign(Object.create(null), defaultOptions, {cwd: process.cwd()}, options);
```

Comment:
I think it was because I removed https://github.com/eslint/eslint/pull/5461/files#diff-3daec82d5218d64d8f2fed4d21f8e6ccL506, so I figured to be save I'd move it up here.  But you're right, it's unnecessary and I'll take it out.